### PR TITLE
DSD-6211 Added new resident changes in release-1.2.3.x

### PR DESCRIPTION
**Pull request by MadhuMosip**

### CODE REVIEW

File: resident-default.properties [hunk 1]
```diff
@@ -727,6 +734,7 @@ resident.template.summary.success.REVOKE_VID=gen-or-revoke-vid-success-summary
 resident.template.summary.success.GET_MY_ID=get-my-uin-card-success-summary
 resident.template.summary.success.VALIDATE_OTP=verify-my-phone-email-success-summary
 resident.template.summary.success.VID_CARD_DOWNLOAD=vid-card-download-positive-summary
+resident.template.summary.success.AUTHENTICATION_REQUEST=mosip.event.type.AUTHENTICATION_REQUEST
```
Comment (vishwa-vyom) [inline resident-default.properties]: why success, simmary and failure has the same value ?

File: resident-default.properties [hunk 2]
```diff
@@ -11,15 +11,20 @@
 logging.level.root=INFO
 logging.level.io.mosip.resident.batch=INFO
 # Whether to enable LoggingFilter - this will print the request details such as URL, headers and body for debugging purpose. Default is false.
-logging.level.io.mosip.resident.config.LoggingInterceptor=INFO
+logging.level.io.mosip.resident.config.LoggingInterceptor=DEBUG
 resident.rest.template.logging.interceptor.filter.enabled=false
 
 resident.logging.filter.enabled=false
-logging.level.io.mosip.resident.filter=INFO
+logging.level.io.mosip.resident.filter=DEBUG
```
Comment (vishwa-vyom) [inline resident-default.properties]: lets revert all the debug logging

File: resident-ui-share-credential-schema.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
 	"identity": [
 		{
-			"attributeName": "name",
+			"attributeName": "fullName",
```
Comment (vishwa-vyom) [inline resident-ui-share-credential-schema.json:4]: why is this change required ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: resident-ui-share-credential-schema.json (hunk 3 above)
Comment (MadhuMosip) [reply resident-ui-share-credential-schema.json:4]: Earlier this property was used as a hardcoded in UI, fullName is the default value coming from API response, using this property directly showing on UI.

File: resident-default.properties (hunk 2 above)
Comment (MadhuMosip) [reply resident-default.properties]: Done

File: resident-default.properties (hunk 1 above)
Comment (MadhuMosip) [reply resident-default.properties]: Changed success, summary and failure values

**Review by vishwa-vyom — APPROVED** [review]
